### PR TITLE
258-make-finishing-touches-on-landing-page

### LINKED ISSUE #258: Make Finishing touches on Landing page
- [x]  Remove Login and SignUp links at bottom of page

- [x]  Add privacy policy, About us and Contact us Links

- [x]  Make Page responsive

- [x]  Add runtime assertions

- [x]  Brush up on styles

- [x]  Using Rollbar for logs

- [x]  Update the Jest tests to reflect changes

### PR DESCRIPTION
**Pull request by glenntanjoh**
Resolves #258 

# Description
Made styling fixes to LandingPage, created a privacy policy component and added tests for all edited files. Added runtime assertions using invariant and used rollbar for logging instead of console logs.

### CODE REVIEW

**Review by Juanan-design — APPROVED** [review]
Good changes

File: frontend/src/components/PrivacyPolicy/PrivacyPolicy.tsx [hunk 1]
```diff
@@ -0,0 +1,219 @@
+import React, { ReactElement } from "react";
+import invariant from "invariant";
+import { useRollbar } from "@rollbar/react";
+import styles from "./PrivacyPolicy.module.css";
+import beapLogo from "../../assets/beap_lab_hex_small.png";
+
+function PrivacyPolicy(): ReactElement {
+    const rollbar = useRollbar();
+    invariant(rollbar, "Rollbar context is not available");
+    rollbar.debug("Reached Privacy Policy page");
```
Comment (cameronbeattie306) [inline frontend/src/components/PrivacyPolicy/PrivacyPolicy.tsx]: Could use rollbar.info

It's the least severe option.

**Review by cameronbeattie306 — APPROVED** [review]
Looks great to me!

File: frontend/src/components/PrivacyPolicy/PrivacyPolicy.tsx (hunk 1 above)
Comment (glenntanjoh) [reply frontend/src/components/PrivacyPolicy/PrivacyPolicy.tsx]: Makes sense. Done

**Pull request merged**